### PR TITLE
My task

### PR DESCRIPTION
**Pull request by kambalas**

### CODE REVIEW

File: ridalot2.0/ridalot2.0/Data/Post.cs [hunk 1]
```diff
@@ -12,5 +12,9 @@ public class Post
         public string? Address { get; set; }
         public int Pay { get; set; }
         public string? Time { get; set; }
+        public string? Worker
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Data/Post.cs:15]: format

File: ridalot2.0/ridalot2.0/Data/Post.cs [hunk 2]
```diff
@@ -12,5 +12,9 @@ public class Post
...
+        {
+            get; set;
```
Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Data/Post.cs:17]: formating

File: ridalot2.0/ridalot2.0/Pages/Feed.razor [hunk 3]
```diff
@@ -52,28 +55,47 @@
 		 OnClose="@ClosePopup"
 		 DialogType="AdPopUp.ModalDialogType.Ok">
 	</AdPopUp>
+}@if (popup1)
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Pages/Feed.razor:58]: more meaningful var name

Comment (neringamaj) [inline ridalot2.0/ridalot2.0/Pages/Feed.razor:58]: format

**Review by Astijq — APPROVED** [review]

**Review by neringamaj — APPROVED** [review]
needs formating

**Pull request merged**